deployment/ccip/sequence/evm/migration/EVM2EVMOnRampToFeeQuoter.go

```diff
@@ -0,0 +1,126 @@
+	m.GasMultiplierWeiPerEth = feetokenCfg.GasMultiplierWeiPerEth
+	m.NetworkFeeUSDCents = feetokenCfg.NetworkFeeUSDCents
+}
+
+func (m *EVM2EVMOnRampMigratePremiumMultiplierCfg) TranslateOnrampToFeeQFeePremiumCfg(token common.Address, feetokenCfg onramp1_5.EVM2EVMOnRampFeeTokenConfig) {
+	m.Token = token
+	m.PremiumMultiplierWeiPerEth = feetokenCfg.GasMultiplierWeiPerEth
+	/* return fee_quoter.FeeQuoterPremiumMultiplierWeiPerEthArgs{
+		Token:                      token,
+		PremiumMultiplierWeiPerEth: feetokenCfg.GasMultiplierWeiPerEth,
+	} */
+}
+
+func (m EVM2EVMOnRampMigrate) TranslateOnrampToFeequoterTokenTransferFeeConfig(token common.Address, onRampTokenTransferFeeConfig onramp1_5.EVM2EVMOnRampTokenTransferFeeConfig) fee_quoter.FeeQuoterTokenTransferFeeConfigSingleTokenArgs {
+	return fee_quoter.FeeQuoterTokenTransferFeeConfigSingleTokenArgs{
+		Token: token,
+		TokenTransferFeeConfig: fee_quoter.FeeQuoterTokenTransferFeeConfig{
+			MinFeeUSDCents:    onRampTokenTransferFeeConfig.MinFeeUSDCents,
+			MaxFeeUSDCents:    onRampTokenTransferFeeConfig.MaxFeeUSDCents,
+			DeciBps:           onRampTokenTransferFeeConfig.DeciBps,
+			DestGasOverhead:   onRampTokenTransferFeeConfig.DestGasOverhead,
+			DestBytesOverhead: onRampTokenTransferFeeConfig.DestBytesOverhead,
+			IsEnabled:         onRampTokenTransferFeeConfig.IsEnabled,
+		},
+	}
+}
+
+// TODO: the below needs to be moved to a common package, cannot right now due to circular dependencies
+const (
+	// https://github.com/smartcontractkit/chainlink/blob/1423e2581e8640d9e5cd06f745c6067bb2893af2/contracts/src/v0.8/ccip/libraries/Internal.sol#L275-L279
+	/*
+				```Solidity
+					// bytes4(keccak256("CCIP ChainFamilySelector EVM"))
+					bytes4 public constant CHAIN_FAMILY_SELECTOR_EVM = 0x2812d52c;
+					// bytes4(keccak256("CCIP ChainFamilySelector SVM"));
+		  		bytes4 public constant CHAIN_FAMILY_SELECTOR_SVM = 0x1e10bdc4;
+				```
+	*/
+	EVMFamilySelector   = "2812d52c"
+	SVMFamilySelector   = "1e10bdc4"
+	AptosFamilySelector = "ac77ffec"
+)
+
+func DefaultFeeQuoterDestChainConfig(configEnabled bool, destChainSelector ...uint64) fee_quoter.FeeQuoterDestChainConfig {
+	familySelector, _ := hex.DecodeString(EVMFamilySelector) // evm
+	if len(destChainSelector) > 0 {
+		destFamily, _ := chain_selectors.GetSelectorFamily(destChainSelector[0])
+		switch destFamily {
+		case chain_selectors.FamilySolana:
+			familySelector, _ = hex.DecodeString(SVMFamilySelector) // solana
+		case chain_selectors.FamilyAptos:
+			familySelector, _ = hex.DecodeString(AptosFamilySelector) // aptos
+		}
+	}
+	return fee_quoter.FeeQuoterDestChainConfig{
+		IsEnabled:                         configEnabled,
+		MaxNumberOfTokensPerMsg:           10,
+		MaxDataBytes:                      30_000,
+		MaxPerMsgGasLimit:                 3_000_000, // TODO: this needs to be updated based on RMN sig verification per chain?! 220/250K
```

Comment:
Needs a little bit more clarification on this on how to distinguish between the messages with RMN sig verification & without